web-console/5-components/order-from-catalog.md

```diff
@@ -54,51 +46,58 @@ There are 3 possible steps in this process: Plan, Configuration and Results.
 
 Post 3.6, an additional Bind Step will be implemented.
 
-### Step 1 - Plan
+### Information Step
 
-### User selects a plan
+![template](img/information-step.png)
 
-#### Catalog item has a single version
-![template](img/plan-step.png)
+1. Catalog Item Icon
+	- metadata.imageURL from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Catalog Item Name
+	- metadata.displayName from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Provider (Optional)
+	- metadata.providerDisplayName from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. List of tags (Optional)
+1. Short description (Optional)
+1. Long description (Optional)
+	- metadata.longDescription from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Image dependencies : Includes an **Image Dependencies** title following by a list of dependencies.  Each image name is preceded by an image icon (note the PF icon)
+1. **View documentation** link which will open a new browser/tab linking to the Documentation URL (Optional)
+	- metadata.documentationUrl from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
 
-#### Catalog item has a multiple versions
+### Plan Step (Optional)
+This step should be shown if there are 1+ plans.
```

Comment:
If a service only has one plan how could a user see plan details? Should they be included on the "information" step in cases where there is only one plan?